client/src/components/SideInfoPanel/SideInfoPanel.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import {
```

Comment:
Nice breaking the line up here - something I need to do on one of my files with all the styled components I have being imported in lol